nuxeo-distribution/nuxeo-functional-tests/src/main/java/org/nuxeo/functionaltests/LogTestWatchman.java

```diff
@@ -166,7 +173,11 @@ protected String getTestClassName(FrameworkMethod method) {
 
     protected void logOnServer(String message) {
         if (driver != null) {
-            if (RestHelper.get("/restAPI/systemLog")) {
+            NuxeoClient client = new NuxeoClientForWebDriver(
+                    new Builder().url(NUXEO_URL).authentication(ADMINISTRATOR, ADMINISTRATOR));
+            Response response = client.get(NUXEO_URL + "/restAPI/systemLog");
+            response.body().close();
+            if (response.isSuccessful()) {
```

Comment:
I made this choice because:
- This class extends a deprecated class from JUnit 4, we should rework it. Also JUnit 5 is here, we might upgrade at some point
- This class is tied to WebDriver and JSF, it will probably be removed soon
- This is the only case where we need to do such a thing, and just for wizard tests because they don't start the REST API
- I didn't want to improve RestHelper, because it is meant to perform requests to REST API as an authenticated user. Here we test the existence of an openUrl resource. We should leverage on something else. 
So, at this time, I didn't want to give more time to develop something here, as the class will probably be removed or refactored.

Reply:
Ok... the last point may make sense, I didn't dig further. 
The other points are not valid to me: you cannot make such assumptions, this is likely wrong and does not take the maintenance into account.